synapse/lib/stormtypes.py

```diff
@@ -7311,6 +7356,9 @@ async def _methViewSet(self, name, valu):
                 self.runt.confirm(('view', 'read'), gateiden=valu)
                 self.runt.confirm(('view', 'fork'), gateiden=valu)
 
+        elif name == 'quorum':
+            valu = await toprim(valu)
+
```

Comment:
odd, i thought that reqValidVdef was used to validate the whole info dict when updating values... :thinking:

Reply:
I thought that too (like in `setViewInfo()`) but only on see it on view creation. Most values are probably fine since they're primitive but `quorum` and `layers` probably should have it. (Although validating `layers` is out of scope for this PR.)

The weird part is validating just `quorum` might require breaking that schema out separately? Ugh.